pkg/atmos/aws-component-helper/suite.go

```
@@ -0,0 +1,177 @@
+
+// Test runs a test within the suite
+func (ts *Suite) Test(t *testing.T, name string, f func(t *testing.T, atm *Atmos)) {
+	if *skipTests {
+		fmt.Printf("Skip test %s/%s\n", ts.name, name)
+		return
+	}
+
+	// Fork the state for the test
+	testState, err := ts.state.Fork(name)
+	require.NoError(t, err)
+	defer testState.Teardown() // Ensure test state is torn down after execution
```

golangci-lint (1.62.2)

Error return value of `testState.Teardown` is not checked

(errcheck)